server/game/cards/07-GR/ReturningChampion.js

```diff
@@ -0,0 +1,40 @@
+const Card = require('../../Card.js');
+
+class ReturningChampion extends Card {
+    // Play: Deal 3 to Returning Champion.
+    //
+    // After Fight: If you are haunted, move all damage from Returning
+    // Champion to another creature.
+    setupCardAbilities(ability) {
+        this.play({
+            gameAction: ability.actions.dealDamage((context) => ({
+                target: context.source,
+                amount: 3
+            }))
+        });
+
+        this.fight({
+            condition: (context) =>
+                context.source.controller.isHaunted() && context.source.hasToken('damage'),
+            target: {
+                activePromptTitle: 'Choose a creature to deal damage to',
```

Comment:
I think "deal damage to" can be confusing. Maybe just say "move damage to"

Reply:
Done.